developerTools/brandColors/css/brandColors.css

```diff
@@ -0,0 +1,76 @@
+.hifi-red-text {
+    color: #ff001a;   
+}
+.hifi-red-background {
+    background-color: #ff001a;       
+}
+
+.hifi-magenta-text {
+    color: #ff42a7;    
+}
+.hifi-magenta-background {
+    background-color: #ff42a7;
+}
+
+.hifi-netural1-text {
```

Comment:
could you please rename these such that users will know what neutral color they're getting? for example, this is a neutral green, so maybe `.hifi-neutral-green-text` ?